src/components/event-details/EventDetails.tsx

```diff
@@ -0,0 +1,119 @@
+import { Button, Col, Row, Typography } from 'antd';
+import dateFormat from 'dateformat';
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import styled from 'styled-components';
+import { EventProps } from '../../containers/upcoming-events/ducks/types';
+const { Title } = Typography;
+
+const TopRow = styled(Row)`
+  margin-bottom: 12px;
+  border: 1px solid #d9d9d9;
+`;
+
+const BottomRow = styled(Row)`
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 12px;
+`;
+
+const CardContent = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+`;
+
+const StyledTitle = styled(Title)`
+  margin-top: 15px;
+`;
+
+const Time = styled.div`
+  margin-bottom: 0.5em;
+`;
+
+const Location = styled.div`
+  margin-bottom: 120px;
+`;
+
+const GreenButton = styled(Button)`
+  color: white;
+  background-color: #2d870d;
+  margin-bottom: 15px;
+`;
+
+const Thumbnail = styled.img`
+  max-width: 100%;
+  max-height: 400px;
+`;
+
+const Info = styled.div`
+  margin-left: 40px;
+  max-height: 350px;
+`;
+
+const AnnoucementBox = styled.div`
+  align-items: center;
+  border: 1px solid #d9d9d9;
+  min-height: 80px;
+`;
+
+const EventListing: React.FC<EventProps> = ({ thumbnail, title, details }) => {
+  const defaultImg =
```

Comment:
This feels like something that should live in a common file, maybe event types? It's referenced a few places.